cli/lib/tasks/state.js

```diff
@@ -52,7 +52,7 @@ const getBinaryDir = (version = util.pkgVersion()) => {
 
 const getVersionDir = (version = util.pkgVersion(), buildInfo = util.pkgBuildInfo()) => {
   if (buildInfo && !buildInfo.stable) {
-    version = ['beta', version, buildInfo.commitBranch, buildInfo.commitSha].join('-')
```

Comment:
I personally like the version being included so you know what you are working with 🤷🏻‍♀️

Reply:
NVM. your above comment clarified.

Since this is just the cache dir, beta could also prob `beta` since the branch name shows its a non-release ver

Reply:
I was thinking that `version` is sorta useless since we use semantic versioning and it can change based on your branch's base commits, but I guess it is still useful as a rough indicator of when the pre-release came out w.r.t Cypress versions. I can add it back in.

Reply:
[d89262e](https://github.com/cypress-io/cypress/pull/20961/commits/d89262e1959359d85f12901c27d674422542de2a)